.env.example

```diff
@@ -1,15 +1,88 @@
-# Fully qualified url your venice next.js app used for redirects, webhooks, background tasks and server-side rendering.
+#################
```

Comment:
@iwasrobbed the `.env.example` file is actually auto-generated based on a schema right now, so these changes could be overwritten next time.

See https://api.codestream.com/c/XkRKwkD2y32n7NEH/ytsQRN3jTziJcwUZ30wd4Q

However maybe we're better off not generating them just yet as it would take a bit of work to update the generation script to produce really nice looking .env example, LMK what you think is best

Reply:
For now let's consider not auto-generating unless it really gets painful to maintain. I'd rather it have super clear docs for when people are setting this up on their own (aka always put the burden on our shoulders if it makes for an exceptional developer experience)